### PR TITLE
feat(forms):Add tests for form component

### PR DESCRIPTION
**Pull request by udhadpa**
Hello All,
Please review the tests written for form component.
It includes tests for css styling as per the redlines.
@JasonLantz @StommePoes @wyseguyonline @mhomolak @umahaea 
Thanks,

### CODE REVIEW

**Comment by umahaea** [comment]
Looks good to me !!

File: src/test/java/elementsTests/FormsTest.java [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+        isMarginBottom = commonUtils.assertValue(marginBottom, expInputsBottomMargin, "Margin-Bottom between Last name and Submit button is not as per the spec");
+        Assert.assertTrue(isMarginBottom);
+
+        // Submit Button
+        submitBtnClass = commonUtils.getAttributeValue(formsPgObj.submitBtn, "class");
+        submitBtnColor = commonUtils.getCSSValue(formsPgObj.submitBtn, "background-color");
+        isSubmitBtnClass = commonUtils.assertValue(submitBtnClass, "pe-btn__cta", "Submit Btn does not inherit from pe-btn__cta ");
+        isSubmitBtnColor = commonUtils.assertCSSProperties("background-color", submitBtnColor, expSubmitBtnColor);
+        if (isSubmitBtnColor == false) {
+            log.info("Background-color of Submit Btn label is not as per spec, actual " + submitBtnColor);
```
Comment (StommePoes) [inline src/test/java/elementsTests/FormsTest.java]: Maybe: "Submit Btn label" -> "Submit Btn text"

File: src/test/java/elementsTests/FormsTest.java [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+        isMarginBottom = commonUtils.assertValue(marginBottom, expInputsBottomMargin, "Margin-Bottom between Last name and Submit button is not as per the spec");
+        Assert.assertTrue(isMarginBottom);
+
+        // Submit Button
+        submitBtnClass = commonUtils.getAttributeValue(formsPgObj.submitBtn, "class", "mobile");
+        submitBtnColor = commonUtils.getCSSValue(formsPgObj.submitBtn, "background-color", "mobile");
+        isSubmitBtnColor = commonUtils.assertCSSProperties("background-color", submitBtnColor, expSubmitBtnColor);
+        isSubmitBtnClass = commonUtils.assertValue(submitBtnClass, "pe-btn__cta", "Submit Btn does not inherit from pe-btn__cta");
+        if (isSubmitBtnColor == false) {
+            log.info("Background-color of Submit Btn label is not as per spec, actual " + submitBtnColor);
```
Comment (StommePoes) [inline src/test/java/elementsTests/FormsTest.java]: same

**Comment by StommePoes** [comment]
I have a remark but looks good

**Comment by udhadpa** [comment]
Hi @StommePoes and @wyseguyonline  : I have made those changes.

**Pull request merged**